### PR TITLE
Inject SVG sprite via ajax

### PR DESCRIPTION
**Pull request by jolheiser**
Per discussion in Discord, `gitea.com` currently has no SVG icons. :scream: 

https://stackoverflow.com/questions/27458729/amazon-s3-cors-issue-with-svg-on-all-major-browser  
https://css-tricks.com/ajaxing-svg-sprite/

This PR uses the method mentioned in the linked post.

Ping @silverwind for feedback or suggestions concerning this method and injecting the sprite into the DOM.

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

**Comment by silverwind** [comment]
Can we inline the content of `/img/svg/icons.svg` into `head.tmpl` right after `<body>`? That'd be my preferred solution as it both circumvents any CORS issues as well as makes the icons immediately available.

**Comment by jolheiser** [comment]
According to the article, this method preserved being able to cache it whereas inlining wouldn't?

**Comment by silverwind** [comment]
Yeah, it would not be able to cache it and it would add 70kB to every HTML request, so probably not the way to go. Ideally, the `<use>` element would expose a `crossorigin` attribute but apparently no browser has implemented such a feature. Testing this right now.

**Comment by silverwind** [comment]
There is a syntax error in where the entry `'{{StaticUrlPrefix}}/img/svg/icons.svg'` is missing a trailing comma, can you fix that here?

**Comment by jolheiser** [comment]
Added the comma

**Comment by silverwind** [comment]
Oh, and please remove serviceworker entries for `'{{StaticUrlPrefix}}/vendor/assets/octicons/octicons.min.css',` and `'{{StaticUrlPrefix}}/vendor/assets/octicons/octicons.woff2?ef21c39f0ca9b1b5116e5eb7ac5eabe6',`, those files no longer exist.

**Comment by silverwind** [comment]
SW working again after those fixes.

**Review by silverwind — APPROVED** [review]

File: web_src/js/index.js [hunk 1]
```diff
@@ -3581,3 +3581,12 @@ window.onOAuthLoginClick = function () {
     oauthNav.show();
   }, 5000);
 };
+
+// AJAX SVGs
```
Comment (silverwind) [inline web_src/js/index.js]:
```suggestion
// Pull SVGs via AJAX to workaround CORS issues with <use> tags
```

**Comment by jolheiser** [comment]
Thanks for the quick response and help! :handshake:

**Comment by silverwind** [comment]
I wasn't aware of this issue with `<use>` not being able to CORS but it's good we have `gitea.com` to test CORS 😆.

**Comment by zeripath** [comment]
Make lg-tm work

**Pull request merged**